frontend/src/js/entity-history/Navigation.tsx

```diff
@@ -104,17 +104,22 @@ export const Navigation = memo(
     }, [entityIds, currentEntityIndex, updateHistorySession]);
 
     const onDownload = useCallback(() => {
-      const idToRow = (entityId: EntityId) => [
-        entityId.id,
-        entityIdsStatus[entityId.id]
-          ? entityIdsStatus[entityId.id].map((o) => o.value)
-          : "",
-      ];
-
-      const csvString = entityIds
-        .map(idToRow)
-        .map((row) => row.join(";"))
-        .join("\n");
+      const usedStatuses = Object.values(entityIdsStatus).reduce(
+        (longest, el) => (longest.length > el.length ? longest : el),
+        [],
+      );
+      const idToRow = (entityId: EntityId) =>
```

Comment:
Why not simply use Papa.unparse for this?

Reply:
yeah good point, done!